### PR TITLE
contrib/vectorscan: new package (5.4.11)

### PR DESCRIPTION
**Pull request by a16bitsysop**

### CODE REVIEW

File: contrib/vectorscan/template.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "-DCMAKE_SKIP_INSTALL_RPATH=ON",
+    "-DBUILD_SHARED_LIBS=ON",
+    "-DFAT_RUNTIME=OFF",
+]
+hostmakedepends = ["cmake", "ninja", "pkgconf", "ragel"]
+makedepends = ["boost-devel", "sqlite-devel"]
+pkgdesc = "High-performance regular expression matching library"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://www.vectorcamp.gr/vectorscan/index.html"
```
Comment (wezm) [inline contrib/vectorscan/template.py:17]: Link redirects to https://www.vectorcamp.gr/ which doesn't seem to describe the library.

Comment (a16bitsysop) [reply contrib/vectorscan/template.py:17]: Okay will find the correct one

File: contrib/vectorscan/template.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+pkgname = "vectorscan"
+pkgver = "5.4.11"
+pkgrel = 0
+archs = ["aarch64", "ppc64", "x86_64"]
```
Comment (q66) [inline contrib/vectorscan/template.py:4]: should probably be ppc64le

(does it not build at all on other platforms?)

File: contrib/vectorscan/template.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+@subpackage("vectorscan-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("vectorscan-progs")
+def _subpkg(self):
+    self.pkgdesc = f"{pkgdesc} (command line tools)"
```
Comment (q66) [inline contrib/vectorscan/template.py:39]: any reason not to use default_progs?

File: contrib/vectorscan/template.py (hunk 1 above)
Comment (a16bitsysop) [reply contrib/vectorscan/template.py:17]: without `index.html` it works, tried `home` and `default` with html and htm extensions as well and all redirected

File: contrib/vectorscan/template.py (hunk 2 above)
Comment (a16bitsysop) [reply contrib/vectorscan/template.py:4]: at the minute no, but I think there will be a software backend in the next release for arch’s with unsupported simd extensions

File: contrib/vectorscan/template.py (hunk 1 above)
Comment (a16bitsysop) [reply contrib/vectorscan/template.py:17]: it works without a `/` on the end of vectorscan as well

File: contrib/vectorscan/template.py (hunk 3 above)
Comment (a16bitsysop) [reply contrib/vectorscan/template.py:39]: no, will try that

**Comment by q66** [comment]
i'll need to manually test this on the other archs

**Comment by a16bitsysop** [comment]
okay

**Comment by q66** [comment]
tests are ok on all archs, however the PR is missing subpackage symlinks (the CI should catch this, but apparently doesn't?)

**Comment by a16bitsysop** [comment]
ah, must have missed adding them.  It fails locally without them.

**Pull request merged**